src/ReverseMarkdown/Converters/Pre.cs

```diff
@@ -17,7 +17,7 @@ public override string Convert(HtmlNode node)
             if (Converter.Config.GithubFlavored)
             {
                 var lang = GetLanguage(node);
-                var code = DecodeHtml(node.InnerText);
+                var code = DecodeHtml(node.InnerText).TrimEnd(new[] {'\r', '\n'});
```

Comment:
Test is failing on TravisCI(*nix), changing `.TrimEnd(new[] {'\r', '\n'})` to `TrimEnd('\r').TrimEnd('\n')` should take care of it.